src/Microsoft.ML.Data/StaticPipe/SchemaBearing.cs

```diff
@@ -37,11 +37,12 @@ private protected SchemaBearing(IHostEnvironment env, StaticSchemaShape shape)
         }
 
         /// <summary>
-        /// Create an object that can be used as the start of a new pipeline, that assumes it uses
-        /// something with the sahape of <typeparamref name="TTupleShape"/> as its input schema shape.
-        /// The returned object is an empty estimator.
+        /// Starts a new pipeline, using the output schema of this object. Note that the returned
+        /// estimator does not contain this object, but it has its schema informed by <typeparamref name="TTupleShape"/>.
+        /// The returned object is an empty estimator, on which a new segment of the pipeline can be created.
         /// </summary>
-        internal Estimator<TTupleShape, TTupleShape, ITransformer> MakeNewEstimator()
+        /// <returns>An empty estimator with the same shape as the object on which it was created</returns>
+        public Estimator<TTupleShape, TTupleShape, ITransformer> MakeNewEstimator()
```

Comment:
>public [](start = 8, length = 6)

As discussed [here](https://github.com/dotnet/machinelearning/pull/809#discussion_r215093150) I believe the POR for right now until we change our minds again is for the method to create new estimator pipes to exist directly on these schema bearing objects (whatever they may be).